modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc

```diff
@@ -51,14 +51,26 @@ ifdef::upstream[]
 +
 [NOTE]
 ====
-If you do not create a default local queue, you must specify a local queue in each pipeline.
+If your cluster administrator does not define a default local queue, you must specify a local queue in each pipeline.
 ====
 endif::[]
 
 * You have access to S3-compatible object storage.
 * You have logged in to {productname-long}.
 * You have created a data science project.
 
+ifndef::upstream[]
+* You have Admin access for the data science project.
+** If you created the project, you automatically have Admin access. 
+** If you did not create the project, the project creator must provide you with Admin access, as described in link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/configuring-access-to-data-science-projects_nb-server[Configuring access to data science projects].
+endif::[]
+ifdef::upstream[]
+* You have Admin access for the data science project.
+** If you created the project, you automatically have Admin access. 
+** If you did not create the project, the project creator must provide you with Admin access, as described in link:{odhdocshome}/working-on-data-science-projects/#_configuring_access_to_data_science_projects[Configuring access to data science projects].
```

Comment:
Same comment as previously, does it need to be the project creator, or anybody with Admin access to the project?